Documentation/gettingstarted/bandwidth-manager.rst

```diff
@@ -174,3 +174,6 @@ Limitations
     * Bandwidth enforcement currently does not work in combination with L7 Cilium Network Policies.
       In case they select the Pod at egress, then the bandwidth enforcement will be disabled for
       those Pods.
+    * Bandwidth enforcement doesn't work with nested network namespace environments like Kind. Because
```

Comment:
Nit: missing main proposal in the sentence.
```suggestion
    * Bandwidth enforcement doesn't work with nested network namespace environments like Kind. This is because
```